server/pom.xml

```diff
@@ -311,6 +311,11 @@
                     <version>${bonita.engine.version}</version>
                     <scope>runtime</scope>
                 </dependency>
+                <dependency>
+                    <groupId>commons-httpclient</groupId>
+                    <artifactId>commons-httpclient</artifactId>
+                    <version>3.1</version>
+                </dependency>
```

Comment:
Why the need for this dependency and isn't the version of this lib already managed in the parent pom. xml ?